index.html

```diff
@@ -200,6 +200,14 @@ <h3>Synchronized rendering of web resources</h3>
           embedded in media containers. Describe a few motivating application
           scenarios.
         </p>
+        <p>
+          During a live media presentation, dynamic and unpredictable events may occur which causes temporary suspension of 
+          the media presentation. During that suspension interval auxiliary content such as the presentation of UI controls a
```

Comment:
This sentence "During that suspension interval auxiliary content such as the presentation of UI controls and media files" doesn't seem to have a predicate (verb). I'm not sure what it means.